.github/workflows/met-web-ci.yml

```diff
@@ -42,8 +42,6 @@ jobs:
       - name: Install dependencies
         run: |
           npm install --legacy-peer-deps
-        env:
-          FONTAWESOME_PACKAGE_TOKEN: ${{ secrets.FONTAWESOME_PACKAGE_TOKEN }}
```

Comment:
It looks like you removed all references to the Fontawesome package token in a later commit. That makes sense if we're just validating for our own Artifactory.